modules/vm/templates/entrypoint.sh.tpl

```diff
@@ -165,7 +165,42 @@ fi
 
 # TODO: -Xmx based on the machine's memory size
 
-# TODO: Backup cron
+# Configure the GraphDB backup cron job
+
+cat <<-EOF > /usr/bin/graphdb_backup
+#!/bin/bash
+
+set -euxo pipefail
+
+az login --identity
+
+BACKUP_NAME="\$(date +'%Y-%m-%d_%H-%M-%S').tar"
+TEMP_BACKUP_DIR="/var/opt/graphdb/"
+RESOURCE_GROUP="\$(az vmss list --query "[0].resourceGroup" --output tsv)"
+STORAGE_ACCOUNT="\$(az storage account list --resource-group "\$RESOURCE_GROUP" --query "[].name" --output tsv)"
```

Comment:
These could fail if this is deployed in existing environment with additional resources (in the future). We could at least filter them by tags?

@yaskoo wdyt

Reply:
They filter by RG, wouldn't the entire Terraform fail if we try to deploy in an existing RG?

Reply:
yes, most likely the filtering here would be the least of our problems if we deploy to an existing environment. We should see if we want to support deploying in an existing RG and address it separately.